contracts/interfaces/USSSpokePoolInterface.sol

```diff
@@ -38,6 +38,28 @@ interface USSSpokePoolInterface {
         int256 payoutAdjustmentPct;
     }
 
+    struct USSRelayerRefundLeaf {
+        // This is the amount to return to the HubPool. This occurs when there is a PoolRebalanceLeaf netSendAmount that
+        // is negative. This is just the negative of this value.
+        uint256 amountToReturn;
+        // Used to verify that this is being executed on the correct destination chainId.
+        uint256 chainId;
+        // This array designates how much each of those addresses should be refunded.
+        uint256[] refundAmounts;
+        // Used as the index in the bitmap to track whether this leaf has been executed or not.
+        uint32 leafId;
+        // The associated L2TokenAddress that these claims apply to.
+        address l2TokenAddress;
+        // Must be same length as refundAmounts and designates each address that must be refunded.
+        address[] refundAddresses;
+        // Merkle tree containing each of the fills refunded by this leaf and accounted for in refundAmounts
+        // and refundAddresses. The MerkleLeaf structure should be defined in the ACROSS-V2 UMIP, as there is no
+        // logic in this contract that supports proving leaf inclusion in this root.
+        bytes32 fillsRefundedRoot;
+        // IPFS hash of the file containing all leaves in the fillsRefundedRoot.
+        string fillsRefundedIpfsHash;
```

Comment:
Do we want to lock this in as an IPFS hash, or could we just call it something more generic like `fillsRefundedHash` ? In principle that might allow us to  swap out the storage layer for something else later on via an UMIP change, without having contrdictory naming.